src/main/java/net/dv8tion/jda/api/interactions/commands/OptionMapping.java

```diff
@@ -342,10 +343,10 @@ public GuildChannel getAsGuildChannel()
      * @return The resolved {@link GuildMessageChannel}, or null if this was not a message channel
      */
     @Nullable
-    public GuildMessageChannel getAsMessageChannel()
+    public GuildMessageChannelUnion getAsMessageChannel()
```

Comment:
Should we get rid of all these specialized getters and just have `getChannel()`?

Reply:
I think maybe yes. We might want to consider having a `ChannelUnion` in that case though.

Reply:
Well in this case GuildChannelUnion, since you can't mention a private channel.

Reply:
Sure